source/clear-linux/get-started/virtual-machine-install/virtualbox.rst

```diff
@@ -21,9 +15,41 @@ Download **version 5.0 or greater** to ensure support for
 the :abbr:`AVX (Advanced Vector Extensions)` needed to run
 |CLOSIA|
```

Comment:
"|CLOSIA|" should only be mentioned once at the beginning.  The rest of the document should just use "|CL|".

Reply:
Recommend changing "VirtualBox is a hypervisor supported by Oracle. You can download it from the
 `official VirtualBox website`_ and select the operating system you are using." to "VirtualBox is a type 2 hypervisor from Oracle.  Download and use version 5.0 or greater from the `official VirtualBox website`. "

Reply:
Done.

Reply:
Answer is yes. Done